OpenRA.Game/Traits/Player/FrozenActorLayer.cs

```diff
@@ -119,6 +125,9 @@ public IEnumerable<IRenderable> Render(WorldRenderer wr)
 				}
 			}
 
+			if (Shrouded)
+				return NoRenderables;
```

Comment:
This is actually a workaround for `FrozenUnderFog`, which will otherwise treat the actor as completely visible(!).